packages/format-currency/test/index.js

```diff
@@ -25,6 +28,19 @@ describe( 'formatCurrency', () => {
 		expect( money ).toBe( '$9,800,900.32' );
 	} );
 
+	test( 'returns trailing zero cents when stripZeros set to true', () => {
+		const money = formatCurrency( 9800900, 'USD', { precision: 2 } );
+		expect( money ).toBe( '$9,800,900.00' );
```

Comment:
Perhaps it'd be nice to also have a test for a currency that has a ',' for a decimal separator.

Reply:
@dzver Thanks! I just added few more test cases for EUR currency as per your suggestion.